.github/workflows/build-docker-images-release.yml

```diff
@@ -35,11 +35,9 @@ jobs:
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
       - name: Build and Push CPU
-        uses: docker/build-push-action@v2
-        with:
-          context: ./docker/accelerate-cpu
-          push: true
-          tags: huggingface/accelerate-cpu:${{needs.get-version.outputs.version}}
+        run: |
+          docker build -t huggingface/accelerate-cpu -f ./docker/accelerate-cpu/Dockerfile .
+          docker push huggingface/accelerate-cpu::${{needs.get-version.outputs.version}}
```

Comment:
Sorry, my docker knowledge is not deep enough: What does the `::` do? `:` would just be tag, right?

Reply:
Yes, small typo, probably from copy/paste

Reply:
(Though didn't seem to cause any issues)